docs/tutorial/Creating-a-Tweet-Field-Type.md

```diff
@@ -1,66 +1,41 @@
-
-<span class="aui-icon aui-icon-small aui-iconfont-approve confluence-information-macro-icon"></span>
-The code created in this tutorial is available on GitHub: <a href="https://github.com/ezsystems/TweetFieldTypeBundle" class="uri" class="external-link">https://github.com/ezsystems/TweetFieldTypeBundle</a>.
+The code created in this tutorial is available on GitHub: (https://github.com/ezsystems/TweetFieldTypeBundle).
 
 This tutorial covers the creation and development of a custom eZ Platform [Field Type](https://doc.ez.no/display/DEVELOPER/Field+Types+reference).
 Field Types are the smallest building blocks of content. eZ Platform comes with about [30 native types](https://doc.ez.no/display/DEVELOPER/Field+Types+reference) that cover most common needs (Text line, Rich text, Email, Author list, Content relation, Map location, Float, etc.)
 
 Field Types are responsible for:
 
--   Storing data, either using the native storage engine mechanisms or specific means
-
--   Validating input data
-
--   Making the data searchable (if applicable)
-
--   Displaying Fields of this type
+- Storing data, either using the native storage engine mechanisms or specific means
+- Validating input data
+- Making the data searchable (if applicable)
+- Displaying Fields of this type
 
 Custom Field Types are a very powerful type of extension, since they allow you to hook deep into the content model.
 
 You can find the in-depth [documentation about Field Types and their best practices here](https://doc.ez.no/display/DEVELOPER/Field+Type+API+and+best+practices). It describes how each component of a Field Type interacts with the various layers of the system, and how to implement those.
 
-Intended audience
------------------
+## Intended audience
 
 This tutorial is aimed at developers who are familiar with eZ Platform and are comfortable with operating in PHP and Symfony2.
 
-Content of the tutorial
------------------------
+## Content of the tutorial
 
 This tutorial will demonstrate how to create a Field Type on the example of a *Tweet* Field Type. It will:
 
--   Accept as input the URL of a tweet (https://twitter.com/&lt;username&gt;/status/&lt;id&gt;)
-
--   Fetch the tweet using the Twitter oEmbed API (<a href="https://dev.twitter.com/docs/embedded-tweets" class="uri" class="external-link">https://dev.twitter.com/docs/embedded-tweets</a>)
+- Accept as input the URL of a tweet (https://twitter.com/&lt;username&gt;/status/&lt;id&gt;)
+- Fetch the tweet using the Twitter oEmbed API (https://dev.twitter.com/docs/embedded-tweets)
+- Store the tweet’s embed contents and URL
+- Display the tweet's embedded version when displaying the field from a template
 
--   Store the tweet’s embed contents and URL
-
--   Display the tweet's embedded version when displaying the field from a template
-
- 
-
-<span class="confluence-embedded-file-wrapper image-center-wrapper"><img src="attachments/31429766/31429765.png" class="confluence-embedded-image image-center" /></span>
```

Comment:
The question is, do we want to stuff the repo with images at this point, until we decide on the ultimate structure for holding docs.